docs/introduction.md

```diff
@@ -0,0 +1,50 @@
+# Introduction
+
+AdvancedHMC.jl provides a robust, modular and efficient implementation of advanced HMC algorithms. 
+AdvancedHMC.jl provides a standalone implementation of NUTS that recovers and extends Stan’s NUTS algorithm. 
+AHMC is written in Julia, a modern high-level language for scientific computing, benefiting from optional 
+hardware acceleration and interoperability with a wealth of existing software written in both Julia and other 
+languages, such as Python.
+
+It is desirable to have a high quality, standalone NUTS implementation in a high-level language,
```

Comment:
I feel the introduction of the paper doesn't necessary fit well in the doc. Maybe merge the two paragraphs and emphasise a few key points
- robust, modular and efficient (this is important for "end" users)
- interoperation with other Julia libs gives us "free" multie-threading, GPU-support, etc.
- well-design modularity is useful for reproducible comparisons (this is important for researchers)

Reply:
Makes sense, I'll fuse the two and emphasise the points more clearly instead of being so verbose.